tests/data/diffusion/test_data_preprocess.py

```diff
@@ -0,0 +1,126 @@
+            'energy': [1.5, 2.5]
+        })
+
+    monkeypatch.setattr(
+        'crystal_diffusion.data.diffusion.data_preprocess.LammpsProcessorForDiffusion.parse_lammps_run',
+        mock_return
+    )
+
+
+def test_prepare_data(mock_processor, mock_parse_lammps_method, tmp_path):
+    # Assuming that the raw_data directory is properly set up with train_run_1 subdirectory
+    train_run_dir = os.path.join(tmp_path, "raw_data", "train_run_1")
+    os.makedirs(train_run_dir)
+    expected_parquet_file = os.path.join(mock_processor.data_dir, "train_run_1.parquet")
+    train_files = mock_processor.prepare_data(os.path.join(tmp_path, "raw_data"), mode='train')
+    assert len(train_files) == 1
+    assert expected_parquet_file in train_files
+
+
+@pytest.fixture
+def mock_parse_lammps_output(monkeypatch):
+    # Create a fake parse_lammps_output function
+    def mock_parse_lammps_output(*args, **kwargs):
+        # Return a fixed DataFrame that imitates the actual parse_lammps_output output
+        return pd.DataFrame({
+            'id': [[1, 2]],
+            'type': [[1, 2]],
+            'x': [[0.1, 0.2]],
+            'y': [[0.2, 0.3]],
+            'z': [[0.3, 0.4]],
+            'fx': [[0.1, -0.1]],
+            'energy': [[1.5, 2.5]],
+            'box': [[1.6, 2.6, 3.6]]
+        })
+    # Use monkeypatch to replace the actual function with the fake one for tests
+    monkeypatch.setattr(
+        'crystal_diffusion.data.diffusion.data_preprocess.parse_lammps_output',
+        mock_parse_lammps_output
+    )
+
+
+def test_parse_lammps_run(mock_processor, mock_parse_lammps_output, tmp_path):
+    # Assuming that the raw_data directory is properly set up with train_run_1 subdirectory
+    train_run_dir = os.path.join(tmp_path, "raw_data", "train_run_1")
+    os.makedirs(train_run_dir)
+    dump_file = os.path.join(train_run_dir, "dump_file")
+    thermo_file = os.path.join(train_run_dir, "thermo_file")
+    Path(dump_file).touch()  # Create the file
+    Path(thermo_file).touch()  # Create the file
+    df = mock_processor.parse_lammps_run(train_run_dir)
+    assert df is not None
+    assert not df.empty
+    assert 'natom' in df.columns
+    assert 'box' in df.columns
+    assert 'type' in df.columns
+    assert 'position' in df.columns
+    assert 'reduced_position' in df.columns
+
+
+@pytest.fixture
+def sample_coordinates():
+    # Sample data frame
+    return pd.DataFrame({
+        'box': [[1, 2, 3]],
+        'x': [[60, 6, 0.6, 0.06]],
+        'y': [[120, 12, 1.2, 0.12]],
+        'z': [[180, 18, 1.8, 0.18]]
+    })
+
+
+def test_convert_coords_to_reduced(sample_coordinates):
+    # Expected output: Each coordinate divided by 10 (the box limit)
+    for index, row in sample_coordinates.iterrows():
+        reduced_coords = LammpsProcessorForDiffusion._convert_coords_to_reduced(row)
+        expected_coords = []
+        for x, y, z in zip(row['x'], row['y'], row['z']):
+            expected_coords.extend([x, y / 2, z / 3])
```

Comment:
This assumes that `box = [1,2,3]`: why not make `box` a fixture  and use it in both `sample_coordinates()` and here?